modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -48,10 +48,13 @@ Default configuration
 === Supported Message Brokers
 
 [%header%autowidth.spread]
-|===
-| Feature | Description | Supported Protocols | Configuration Details 
-| Anypoint MQ Message Broker | Supports asynchronous messaging with Anypoint MQ. | anypointmq | Anypoint MQ configurations are specified in the server object of the AsyncAPI specification. Supports both `Publish` and `Subscribe` operations. 
-| Kafka Message Broker | Supports asynchronous messaging with Apache Kafka. | kafka, kafka-secure | Kafka configurations are specified in the server object of the AsyncAPI specification. Supports both `Publish` and `Subscribe` operations. 
+
+|=== 
+| Message Broker | Description | Supported Protocols | Configuration Details 
+| Anypoint MQ | Supports asynchronous messaging with Anypoint MQ. | anypointmq | Anypoint MQ configurations are specified in the server object of the AsyncAPI specification. Supports both `publish` and `subscribe` operations. 
```

Comment:
Perhaps we can avoid using terminology from AsyncAPI specification here and refer APIKit operations instead. Reason is, going forward AsyncAPI 3.0 will have operations `send` and `receive`; when we support 3.0, this section should include them too. 
Thus, instead of `publish` and `subscribe` we can refer to `Message Listener` and `Publish`

Reply:
Hi, @aby-philip-sf!
We can work on the best wording for referring to the `subscribe` and `publish` operations after the Oct release. Most documents in ACB, APIkit, Studio, and other products use the `subscribe` and `publish` terms. 
According to my understanding, if we make the change in this APIkit for AsycnAPI doc, we should make the change in the other docs, right?
Thanks for your support!

Reply:
Hi @Crispy-Salesforce , here is how i see this
- _APIKit for Async API_ provides an implementation of _Async API_ specification (of which we support till 2.6 version)
- As part of this, _APIKit for Async API_ supports multiple protocols (e.g. kafka, Anypoint MQ etc). 
- Protocol support does not vary based on _Async API_ version. i.e. we do not have a case of kafka supporting _Async API_ 2.6 but not 3.0. 
- This page is about which protocols we support. 

Hence my reasoning that this page can refer to just _APIKit for Async API_ instead of _Async API_ specification. In other docs, we are discussing details of the _APIKit for Async API_ implementation of _Async API_ specification and should indeed refer to terms like `publish` and `subscribe`. They should not be affected